File: server/sod/app/appSettings/appSettings.go
/*
 * Copyright (C) The Qing Project. All rights reserved.
 *
 * Use of this source code is governed by a license that
 * can be found in the LICENSE file.
 */

package appSettings

type AppSettings struct {
	Community CommunitySettings `json:"community"`
}

type CommunitySettings struct {
	QueAndDis          bool `json:"queAndDis"`
	ForumsEnabled      bool `json:"forumsEnabled"`
	ForumGroupsEnabled bool `json:"forumGroupsEnabled"`
}

File: web/src/sod/app/appSettings.ts
/*
 * Copyright (C) The Qing Project. All rights reserved.
 *
 * Use of this source code is governed by a license that
 * can be found in the LICENSE file.
 */

export interface AppSettings {
  community?: CommunitySettings;
}

export interface CommunitySettings {
  queAndDis?: boolean;
  forumsEnabled?: boolean;
  forumGroupsEnabled?: boolean;
}
